### ISSUE TITLE
AppArmor complaining about QEMU

### ISSUE THREAD

**Issue by erhudy** [bloomberg/chef-bcpc#743]
This gets spewed into `dmesg` on the regular on running hypervisors:
```
[83768.847824] type=1400 audit(1438180376.667:12391): apparmor="DENIED" operation="open" \
profile="libvirt-1c01664c-8f99-4942-85c9-2c0c90b2e92b" name="/run/shm/lttng-ust-wait-5" \
pid=1104 comm="qemu-system-x86" requested_mask="r" denied_mask="r" fsuid=112 ouid=0
```

**Comment by caiush** [comment]
yeah this is a known issue, solution isnt clear to me atm as we are already using hammer:
https://bugs.launchpad.net/ubuntu/+source/libvirt/+bug/1432644

**Comment by erhudy** [comment]
Maybe it was only patched in Canonical's version of Hammer, since we switched back to the Inktank packages for Hammer.

**Comment by erhudy** [comment]
This is fixed as of Ceph 0.94.6, which we are now running everywhere: http://tracker.ceph.com/issues/13274
